crates/core/src/rlp/encode.rs

```diff
@@ -279,6 +279,14 @@ impl RLPEncode for ethereum_types::H512 {
     }
 }
 
+impl RLPEncode for ethereum_types::U256 {
+    fn encode(&self, buf: &mut dyn BufMut) {
+        let mut tmp_buf = vec![];
+        self.to_big_endian(&mut tmp_buf);
+        tmp_buf.encode(buf)
+    }
+}
```

Comment:
@ricomateo already implemented this in #60, and also found the encoded U256 can have a length of less than 32 bytes, depending on the amount of leading zeros.

Reply:
Left his implementation when updating with main branch